tools/swift/duckdb-swift/Sources/DuckDB/Types/Time.swift

```diff
@@ -69,3 +69,8 @@ private extension Time.Components {
     self = ctimestruct.asTimeComponents
   }
 }
+
+public struct TimeTz: Hashable, Equatable, Sendable {
```

Comment:
Not important now but for future reference, TIME_TZ is actually orderable, but the rules are not obvious (its a two field compare with time as the major sort and -offset as the minor).

Reply:
So something like:

```swift
public static func < (lhs: TimeTz, rhs: TimeTz) -> Bool {
  (lhs.time, -lhs.offset) < (rhs.time, -rhs.offset)
}
```
?

Reply:
Yeah I think so.